Resources/public/templates/fields/edit/relationlist.hbt

```diff
@@ -0,0 +1,45 @@
+<div class="pure-g ez-editfield-row ez-asynchronousview">
+    <div class="pure-u ez-editfield-infos">
+        {{> ez_fieldinfo_tooltip }}
+        <label for="ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}">
+            <p class="ez-fielddefinition-name">
+                {{ fieldDefinition.names.[eng-GB] }}{{#if isRequired}}*{{/if}}:
+            </p>
+            <p class="ez-editfield-error-message">&nbsp;</p>
+        </label>
+    </div>
+    <div class="pure-u ez-editfield-input-area ez-default-error-style">
+        <div class="ez-editfield-input"><div class="ez-relation-input-ui">
+            {{#if isEmpty}}
+                <p class="ez-relation-empty">No relation.</p>
+            {{else}}
+                {{#if destinationContents }}
+                <table border="1">
```

Comment:
remove the `border="1"`, this should be handled in the CSS. For a nicer table, you can also use the PureCSS style http://purecss.io/tables/ (it's used in https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/views/Section/list.html.twig#L54 for instance)